cmd/mdatagen/internal/sampleconnector/metadata.yaml

```diff
@@ -0,0 +1,155 @@
+# Sample metadata file with all available configurations for a connector.
+
+type: sample
+github_project: open-telemetry/opentelemetry-collector
+
+sem_conv_version: 1.9.0
+
+status:
+  class: connector
+  stability:
+    development: [metrics_to_metrics]
+  distributions: []
+  unsupported_platforms: [freebsd, illumos]
+  codeowners:
+    active: []
+  warnings:
+    - Any additional information that should be brought to the consumer's attention
+
+resource_attributes:
+  string.resource.attr:
+    description: Resource attribute with any string value.
+    type: string
+    enabled: true
+
+  string.enum.resource.attr:
+    description: Resource attribute with a known set of string values.
+    type: string
+    enum: [one, two]
+    enabled: true
+
+  optional.resource.attr:
+    description: Explicitly disabled ResourceAttribute.
+    type: string
+    enabled: false
+
+  slice.resource.attr:
+    description: Resource attribute with a slice value.
+    type: slice
+    enabled: true
+
+  map.resource.attr:
+    description: Resource attribute with a map value.
+    type: map
+    enabled: true
+
+  string.resource.attr_disable_warning:
+    description: Resource attribute with any string value.
+    type: string
+    enabled: true
+    warnings:
+      if_enabled_not_set: This resource_attribute will be disabled by default soon.
+
+  string.resource.attr_remove_warning:
+    description: Resource attribute with any string value.
+    type: string
+    enabled: false
+    warnings:
+      if_configured: This resource_attribute is deprecated and will be removed soon.
+
+  string.resource.attr_to_be_removed:
+    description: Resource attribute with any string value.
+    type: string
+    enabled: true
+    warnings:
+      if_enabled: This resource_attribute is deprecated and will be removed soon.
+
+attributes:
+  string_attr:
+    description: Attribute with any string value.
+    type: string
+
+  overridden_int_attr:
+    name_override: state
+    description: Integer attribute with overridden name.
+    type: int
+
+  enum_attr:
+    description: Attribute with a known set of string values.
+    type: string
+    enum: [red, green, blue]
+
+  boolean_attr:
+    description: Attribute with a boolean value.
+    type: bool
+
+  # This 2nd boolean attribute allows us to test both values for boolean attributes,
+  # as test values are based on the parity of the attribute name length.
+  boolean_attr2:
+    description: Another attribute with a boolean value.
+    type: bool
+
+  slice_attr:
+    description: Attribute with a slice value.
+    type: slice
+
+  map_attr:
+    description: Attribute with a map value.
+    type: map
+
+metrics:
```

Comment:
Why is this added to a connector? The `metrics` section is supposed to be set on scraping receivers only

Reply:
Sorry, I missed the discussion above... Do you have a connector emitting metrics open sourced? It's in contrib, right?

Reply:
> Sorry, I missed the discussion above... Do you have a connector emitting metrics open sourced? It's not in contrib, right?

Right